ipfs-api-prelude/src/lib.rs

```diff
@@ -16,12 +16,18 @@ mod api;
 mod backend;
 mod error;
 mod from_uri;
+mod global_opts;
 mod header;
 mod read;
 pub mod request;
 pub mod response;
 
 pub use {
-    api::IpfsApi, backend::Backend, error::Error, from_uri::TryFromUri, request::ApiRequest,
+    api::IpfsApi,
+    backend::Backend,
+    error::Error,
+    from_uri::TryFromUri,
+    global_opts::{BackendWithGlobalOptions, GlobalOptions},
```

Comment:
Can we export this in `ipfs-api-backend-hyper` and `ipfs-api-backend-actix` so people don't need to declare `ipfs-api-prelude` as a dependency (looks like only `GlobalOptions` is exported)?